Editor/AGS.Editor/DataFileWriter.cs

```diff
@@ -1282,6 +1282,13 @@ public static bool SaveThisGameToFile(string fileName, Game game, CompileMessage
             writer.Write(NativeConstants.GAME_DATA_VERSION_CURRENT);
             writer.Write(AGS.Types.Version.AGS_EDITOR_VERSION.Length);
             WriteString(AGS.Types.Version.AGS_EDITOR_VERSION, AGS.Types.Version.AGS_EDITOR_VERSION.Length, writer);
+            // Write extended engine caps; none for this version
```

Comment:
Would it be worth setting it up so all non-release builds have a special CAP flag?